package.json

```diff
@@ -43,7 +43,7 @@
     "prepare_release:sdk:patch": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:version_patch && npm run _restore:package-json && npm run _changelog:prepare_experimental && npm run _changelog:prepare_stable",
     "prepare_release:sdk:minor": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:version_minor && npm run _restore:package-json && npm run _changelog:prepare_experimental && npm run _changelog:prepare_stable",
     "prepare_release:all:minor": "npm run _check:no_changes && npm run _backup:package-json && npm run _lerna:remove_api && npm run _lerna:version_minor && cd api/ && npm version minor && cd .. && lerna run align-api-deps && npm run _restore:package-json && npm run _changelog:prepare_all",
-    "release:publish": "lerna publish from-package --no-push --no-private --no-git-tag-version --no-verify-access",
+    "release:publish": "lerna publish --concurrency 1 from-package --no-push --no-private --no-git-tag-version --no-verify-access",
```

Comment:
Reviewer note: I've had trouble publishing without limiting concurrency in the  past. Sometimes, it tries to publish a package while a dependency is not available yet and exits with an error. Having `--concurrency 1` will take slightly longer but will avoid running into a state where only a portion of the packages are published.